.github/workflows/prepare_deployment_branch.yaml

```diff
@@ -27,13 +27,19 @@ jobs:
           echo "Branch name: \"${BRANCH_NAME}\""
 
       - name: "Create branch '${{ env.BRANCH_NAME }}' to contain the changes for the deployment on ${{ env.DEPLOYMENT_DATE }}"
+
         uses: JosiahSiegel/remote-branch-action@dbe7a2138eb064fbfdb980abee918091a7501fbe
+        ## emvaldes@hotmail.com (Replace) - uses: ./.github/actions/remote-branch-action
+
         with:
           branch: "${{ env.BRANCH_NAME }}"
 
       - name: "Prepare a Pull Request from ${{ env.BRANCH_NAME }} into production branch"
         id: pr
+
         uses: JosiahSiegel/reliable-pull-request-action@ae8d0c88126329ee363a35392793d0bc94cb82e7
+        ## emvaldes@hotmail.com (Replace) - uses: ./.github/actions/reliable-pull-request-action
```

Comment:
corp email please

Reply:
I have updated the email address to reflect the Aquia corporate email.